change/@fluentui-dom-utilities-3c9941d3-fd1e-4a07-85f3-e239836f2cfc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "chore: disallow document and window access",
```

Comment:
tip for future: chore != patch, this added new api on public surface while refactoring under the hood. it should have been type:minor and `feat`